external-crates/move/crates/move-compiler/tests/move_2024/typing/all_deprecation_attributes.exp

```diff
@@ -0,0 +1,118 @@
+warning[W04041]: use of deprecated function
+   ┌─ tests/move_2024/typing/all_deprecation_attributes.move:16:9
+   │
+16 │         internal();
+   │         ^^^^^^^^ This function is deprecated: This is a deprecated function within a deprecated module.
+
+warning[W04038]: use of deprecated module
+   ┌─ tests/move_2024/typing/all_deprecation_attributes.move:31:12
+   │
+31 │         f: X,
+   │            ^ This module member is deprecated since the whole module is marked deprecated: Use the k module instead.
```

Comment:
How hard is it to determine if this is a struct or enum to make this language more-consistent?